IaC/PnP/monitoring/aks/online/aks_secure_baseline/configuration/monitor/diagnostics.tfvars

```diff
@@ -0,0 +1,35 @@
+diagnostics_definition = {
```

Comment:
We should create a monitoring task for understanding all of these diagnostic streams and how they can be used

Reply:
@atxryan Can you capture this in an issue?